src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -273,19 +274,31 @@ function lockedUntil(lockEndDate?: number) {
       </template>
       <template #iconColumnCell="pool">
         <div v-if="!isLoading" class="py-4 px-6">
-          <BalAssetSet :addresses="orderedTokenAddresses(pool)" :width="100" />
+          <BalAssetSet
+            :addresses="
+              POOLS.Names[pool.id]
+                ? [pool.address]
+                : orderedTokenAddresses(pool)
+            "
```

Comment:
Can we maybe make these addresses a computed property and do a check to make sure we have an icon for a pool.address first before we use it, otherwise just default to token addresses.

Reply:
Do you mean add new field `iconAddresses` to `Pool` type? Or move this logic to a method? It's not possible to have this line as a computed property within `PoolsTable`.

Reply:
Ahh right, yeah it can't be a computed property but we could have a method like:
```ts
function iconAddresses(pool) {
  // Check if we have a logoURI for pool address, if so return pool address
  // else use pool token addresses
}
Again, let's not add this to the Pool type just keep it contained in the PoolsTable.

Reply:
You will need to use the useToken composable again to make this function work.

Reply:
It's a non-trivial task to check if we have the icon for the `pool.address` - as token information sometimes includes `logoURI` which returns 404 - e.g. for AuraBAL: "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x3dd0843A028C86e0b760b1A76929d1C5Ef93a2dd/logo.png"

Maybe instead, we should try to ensure that all the pools with friendly names definitely have the correct logoURIs?